src/dymaptic.GeoBlazor.Core/Scripts/dotNetBuilder.ts

```diff
@@ -564,10 +567,9 @@ function buildDotNetViewHit(viewHit: ViewHit): DotNetViewHit | null {
             return {
                 type: "graphic",
                 graphic: buildDotNetGraphic(viewHit.graphic),
-                layer: buildDotNetLayer(viewHit.layer ?? viewHit.graphic.layer),
+                layer: buildDotNetLayer(viewHit.layer ?? viewHit.graphic.layer, false),
```

Comment:
setting false on `buildDotNetLayer` will limit the return to just the graphic layer. true would enable the complete layer and all graphics to be returned